config/jobs/kubernetes/sig-cloud-provider/gcp/sig-gcp-gce-config.yaml

```diff
@@ -536,5 +536,5 @@ periodics:
       - --up=false
       image: gcr.io/k8s-testimages/kubekins-e2e:v20190812-5a136da-master
   annotations:
-    testgrid-dashboards: sig-release-1.12-all
+    testgrid-dashboards: sig-release-1.13-all
```

Comment:
since I can't comment on unchanged lines

you missed some jobs in here
- ci-kubernetes-soak-gci-gce-beta is pointed at 1.15, should be 1.16
- ci-kubernetes-soak-gci-gce-stable1 is pointed at 1.14, should be 1.15
- ci-kubernetes-soak-gci-gce-stable2 is pointed at 1.13, should be 1.14

Reply:
I explicitly only made attempts to clean up the jobs that would block this PR from merging.
It's a larger problem, detailed [here](https://github.com/kubernetes/test-infra/issues/13883) and discussed [here](https://kubernetes.slack.com/archives/CJH2GBF7Y/p1565741344156500?thread_ts=1565740529.154900&cid=CJH2GBF7Y).